.swiftformat

```diff
@@ -0,0 +1,8 @@
+--extensionacl on-declarations
+--redundanttype explicit
+--swiftversion 5.10
+--maxwidth 120
+--header "{file}\nSwiftyBeaver\n\nCopyright (c) 2015 Sebastian Kreutzberger\nAll rights reserved.\n\nCopyright 2025 MFB Technologies, Inc.\n\nThis source code is licensed under the MIT License (MIT) found in the\nLICENSE file in the root directory of this source tree."
```

Comment:
You could just eliminate this header. I think for OSS projects a header in every file is overkill.

Reply:
Made this change as I understood it.